.travis.yml

```diff
@@ -39,8 +39,12 @@ before_install:
       export GCC_COLORS='error=01;31:warning=01;35:note=01;36:caret=01;32:locus=01:quote=01';
     fi;
   # Add coverall for C++ so coverall.io could be triggered. Even it should be --coverage and gcov.
+  # Install updated libglew-dev since the version provided by trusty is outdated
```

Comment:
The proposed changes don't change the hardware requirement. Previously the GL renderer was unconditionally using some extensions, that aren't part of any official OpenGL standard, apart from being specified as optional extensions. Those extensions were later revised and included officially in the 4.3 and 4.5 standards respectively, I think. They can be implemented on prior hardware, however in the open source OpenGL implementation (mesa) on linux the developers chose to only implement the current versions, that are also part of OpenGL 4.5.
The proposed changes only add proper checks if the required extensions are supported and fallbacks to use the ARB extensions if the EXT versions aren't available.
This can only increase hardware support as in the case the EXT versions of the extensions are supported, nothing changes. On linux this change allows rpcs3 to actually render anything on the screen if you are using mesa as it falls back to the ARB versions.

The newer GLEW is needed, as only since GLEW 1.11 the header defines the necessary prototypes and defines to be able to check if a driver supports those functions. The functions can be unsupported by the driver, the newer GLEW version only allows checking for them and if they are supported, loading them.